### DISCUSSION TITLE
FFMPEG problem

### DISCUSSION THREAD

**Discussion by moneydude242** [eaglerforge/EaglerForge#21] [Q&A] [answered]
I was trying to get download the newest Eaglerforge Client. However, whenever I reach the part where I am required to download the FFMPEG and add the path, I cannot. I downloaded the exe file from [https://www.gyan.dev/ffmpeg/builds/](gyan.dev), and I extracted the files. However, whenever I click on the folder where ffmpeg.exe is located, it returns "Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException: Cannot invoke "java.awt.Image.getWidth(java.awt.image.ImageObserver)" because "retVal" is null",  and a whole bunch of locations (I am relatively new to java, so I have no experience with this.). Could anybody help with this issue? Thanks!

**Comment by leahanderson1** [comment, marked as answer]
https://github.com/EaglerForge/EaglerForge-builds if the compiler is having troubles.
also, this seems like an eagler in general problem rather than an eaglerforge-specific one.

**Comment by TheKevinEpic** [comment]
when i compile eaglercraft in general i just put the .exe file (u dont need it added to ur path)

**Comment by stpv221** [comment]
From the bin, right?
